tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -255,6 +255,15 @@ def test_nan_to_num_broadcast(self, kwarg):
             with pytest.raises(ValueError):
                 xp.nan_to_num(0.0, **{kwarg: y})
 
+    def test_real_if_close(self):
+        x = cupy.random.rand(10) + 1e-15j
+        assert cupy.all(cupy.real_if_close(x) == numpy.real_if_close(x))
+
+    def test_real_if_close_with_tol(self):
+        x = cupy.random.rand(10) + 1e-7j
+        assert cupy.all(cupy.real_if_close(x, tol=1e-6)
+                        == numpy.real_if_close(x, tol=1e-6))
+
```

Comment:
Could you add tests for inputs whose imaginary part is greater than tolerance?

Reply:
Edit the test or add a new one?

Reply:
Could you add a new one?